analytics/population/nc_pop_recent.ipynb

```diff
@@ -0,0 +1,2597 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.        column= 'NAME', # make choropleth based on 'NC name' column
</pre><p>nit: for inline comments: two spaces between code and "#"; please capitalize start of all comments and put a period at the end of all comments.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/hackforla/311-data/pull/1336/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>Made all the changes. </p>